src/OrchardCore.Modules/OrchardCore.Media/Assets/scss/_main-layout.scss

```diff
@@ -55,6 +62,8 @@
             height: 100%;
             width: 240px;
             overflow: auto;
+            scrollbar-width: thin;
+            scrollbar-color: #495057 #EAEAEA;
```

Comment:
Should we do these with bootstrap variables? 
or just variables in the variables file (which are copied from bootstrap from memory)

Reply:
More on this. These styles will work with Firefox only. We use now `::-webkit-scrollbar` for Chrome based browser. It won't work with Internet Explorer but it will still render an unstyled scrollbar. I don't think we should matter about it as now the new Edge (Chromium) is out.

Reply:
I will add that the perfect-scrollbar was buggy. It was not retaining the proper height of elements all the time because we use absolute and relative positioning in that menu. I had no choice to remove it so that it works correctly.